src/util.c

```diff
@@ -468,3 +469,15 @@ const char *pga_details(const PgAddr *a, char *dst, int dstlen)
 	}
 	return dst;
 }
+
+/*
+ * parser setter for parsing auth_dbname parameter using "cfparser"
+ */
+bool cf_set_authdb(struct CfValue *cv, const char *value)
+{
+    if (value && strcmp(value, "pgbouncer") == 0) {
```

Comment:
we make this "pgbouncer" check in multiple places. Can we move it to a shared place, e,g., `check_reserved_database`

Reply:
Great point, I will do it soon